src/components/project/sections/the-challenge.tsx

```diff
@@ -1,27 +1,197 @@
+const StyledImage = styled(motion(Image))`
+  border-radius: 26px;
+  object-fit: cover;
+  width: 375px;
+  height: 774px;
+  position: absolute;
+  @media (min-width: 1350px) {
+    top: 92px;
+    left: 220px;
+  }
+
+  @media (min-width: 1088px) and (max-width: 1350px) {
+    top: 92px;
+    left: 138px;
+  }
+
+  @media (max-width: 1088px) {
+    top: 92px;
+    left: 75px;
+  }
+`;
+const StyledMobileImageWrapper = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: scroll;
+  gap: 8px;
+  width: 100%;
+  scrollbar-width: none;
+`;
+const StyledMobileImage = styled(Image)`
+  border-radius: 26px;
+  object-fit: cover;
+  width: 212px;
+  height: 393px;
+  flex-shrink: 0;
+
+  @media (max-width: ${breakpoints.sm}px) {
+    width: 162px;
+    height: 343px;
+  }
+`;
+
+const StyledParagraphImage = styled.span`
+  color: ${colors.accent};
+  transition: color 0.2s;
+
+  &:hover {
+    color: ${colors.white};
+  }
+  & > span {
+    text-decoration: underline;
+  }
+`;
+const StyledParagraph = styled.p`
+  font-size: ${getRemSize(dimensions.textSizes.large.desktop)};
+  line-height: 1.15;
+  letter-spacing: 2px;
+  font-weight: 500;
+`;
+
+const StyledMobileParagraph = styled.p`
+  font-size: ${getRemSize(dimensions.textSizes.normal.desktop)};
+  line-height: 1.3;
+  letter-spacing: 1px;
+  font-weight: 400;
+  text-indent: 72px;
+  margin-bottom: 40px;
+`;
+const StyledMobileParagraphImage = styled.span`
+  font-size: ${getRemSize(dimensions.textSizes.normal.desktop)};
+`;
 export default function DynamicTextAndImages({ content }: IProps) {
+  const isDesktop = useIsDesktop();
+  const isTablet = useIsTablet();
+  const [hoverIndex, setHoverIndex] = useState(null);
+  const onHover = (index) => {
+    setHoverIndex(index);
+  };
+
+  const onHoverOver = () => {
+    setHoverIndex(null);
+  };
   return (
-    <>
-      {content.map((item, index) => {
-        <p>{item.text}</p>;
-        {
-          item.image && (
-            <Image
-              key={index}
-              width={500}
-              height={200}
-              alt={`Gallery Image ${index}`}
-              src={item.image.node.sourceUrl}
-            />
-          );
-        }
-      })}
-    </>
+    <StyledBigWrapper>
+      <Row>
+        {isDesktop ? (
+          <>
+            <Col start={1} span={4}>
+              <StyledPillWrapper>
+                <Pill pillText="The challenge" />
+              </StyledPillWrapper>
+            </Col>
+            <Col start={5} span={8}>
+              <StyledWrapper>
+                <StyledParagraph>
+                  {content.map((item, index) => {
+                    return item?.image ? (
+                      <StyledParagraphImage
+                        onMouseEnter={(e) => onHover(index)}
+                        onMouseLeave={(e) => onHoverOver()}
+                      >
+                        {" "}
+                        <span>{item.text}</span>
+                      </StyledParagraphImage>
```

Comment:
We need a space here because on mobile screens this word is stuck to the previous word

```suggestion
                        <span>{" " + item.text}</span>
```

Reply:
I tested it on desktop and the space doesn't appear for me, maybe just double check